tests/unit/email/test_services.py

```diff
@@ -101,10 +101,18 @@ def test_creates_service(self):
         assert sender._sender == "DevPyPI <noreply@example.com>"
         assert sender._db is request.db
 
-    def test_send(self, db_session):
+    def test_send(self, db_session, monkeypatch):
```

Comment:
it's not clear to me what the inclusion of the `monkeypatch` fixture is doing here, resetting the touch of `random.seed`?

Reply:
Oops, nope, just accidently leftover.